migrations/versions/27833deaf81f_add_idp.py

```diff
@@ -0,0 +1,26 @@
+"""Add IDP
+
+Revision ID: 27833deaf81f
+Revises: a38a346e6ded
+Create Date: 2020-03-15 19:38:26.321139
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "27833deaf81f"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
you should add an initial migration that creates all the existing tables so that someone can just use alembic to get the db ready

Reply:
i did write an initial migration but it would break when trying to create tables that already exist (in existing deployments), so i removed it. i think the model with `idp` has to be considered the initial state